src/FrontDoor/FrontDoor/help/Get-AzFrontDoorFireWallPolicy.md

```diff
@@ -22,55 +22,61 @@ The **Get-AzFrontDoorFireWallPolicy** cmdletGet gets WAF policy in a resource gr
 
 ## EXAMPLES
 
-### Example 1: Get a WAF policy called $Name in $resourceGroup
+### Example 1
 ```powershell
-PS C:\> Get-AzFrontDoorFireWallPolicy -Name $Name -ResourceGroupName $resourceGroup
-
-PolicyMode         : Prevention
-PolicyEnabledState : Enabled
-CustomRules        : {Rule1}
-ManagedRules       : {Microsoft.Azure.Commands.FrontDoor.Models.PSAzureManagedRule}
-Etag               :
-ProvisioningState  : Succeeded
-Tags               :
-Id                 : /subscriptions/{subid}/resourcegroups/{resourceGroupName}/providers/Micr
-                     osoft.Network/frontdoorwebapplicationfirewallpolicies/{Name}
-Name               : {Name}
-Type               :
+PS C:\> Get-AzFrontDoorFireWallPolicy -Name $policyName -ResourceGroupName $resourceGroupName
+
+PolicyMode                    : Prevention
```

Comment:
Default display should be table format, see: https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/generate-format.ps1xml-file.md

Reply:
Changed help to show a table.